util/config.go

```diff
@@ -119,6 +119,11 @@ func (cfg *Config) WithTokens(accessTokenType, accessToken, refreshToken, idToke
 	return cfg
 }
 
+/* Return the ID token, or "" if does not exist. */
+func (cfg *Config) GetIdToken() string {
```

Comment:
On second thought. I think the config code should just have an options map for each target. There is no need for the config code to know anything about tokens, and we have to change it every time we add something new (like saving the public key), and getting fields is convoluted due to the current target indirection, so I think we should just change the config code to return a map of options for current target.